project_multifamily_beta/housing_characteristics/Insulation Wall.tsv

```diff
@@ -1,91 +1,91 @@
-Dependency=Location Region	Dependency=Vintage	"Option=Wood Stud, Uninsulated"	"Option=Wood Stud, R-7"	"Option=Wood Stud, R-11"	"Option=Wood Stud, R-15"	"Option=Wood Stud, R-19"
-CR02	<1940	0.31	0.17	0.31	0.14	0.07
-CR05	<1940	1	0	0	0	0
-CR03	<1940	1	0	0	0	0
-CR04	<1940	1	0	0	0	0
-CR06	<1940	1	0	0	0	0
-CR07	<1940	1	0	0	0	0
-CR08	<1940	1	0	0	0	0
-CR09	<1940	1	0	0	0	0
-CR10	<1940	1	0	0	0	0
-CR11	<1940	1	0	0	0	0
-CR02	1950s	0.09	0.32	0.45	0.13	0
-CR05	1950s	1	0	0	0	0
-CR03	1950s	1	0	0	0	0
-CR04	1950s	1	0	0	0	0
-CR06	1950s	1	0	0	0	0
-CR07	1950s	1	0	0	0	0
-CR08	1950s	1	0	0	0	0
-CR09	1950s	1	0	0	0	0
-CR10	1950s	1	0	0	0	0
-CR11	1950s	1	0	0	0	0
-CR02	1960s	0.06	0.15	0.44	0.24	0.12
-CR05	1960s	1	0	0	0	0
-CR03	1960s	1	0	0	0	0
-CR04	1960s	1	0	0	0	0
-CR06	1960s	1	0	0	0	0
-CR07	1960s	1	0	0	0	0
-CR08	1960s	1	0	0	0	0
-CR09	1960s	1	0	0	0	0
-CR10	1960s	1	0	0	0	0
-CR11	1960s	1	0	0	0	0
-CR02	1970s	0	0.06	0.71	0.12	0.1
-CR05	1970s	0	1	0	0	0
-CR03	1970s	0	1	0	0	0
-CR04	1970s	0	1	0	0	0
-CR06	1970s	0.5	0.5	0	0	0
-CR07	1970s	0	1	0	0	0
-CR08	1970s	0.5	0.5	0	0	0
-CR09	1970s	0.5	0.5	0	0	0
-CR10	1970s	0.5	0.5	0	0	0
-CR11	1970s	0.5	0.5	0	0	0
-CR02	1980s	0	0.01	0.27	0.1	0.63
-CR05	1980s	0.02	0.07	0.83	0	0.08
-CR03	1980s	0.01	0.02	0.57	0	0.41
-CR04	1980s	0.01	0.01	0.82	0	0.17
-CR06	1980s	0.01	0.02	0.9	0	0.06
-CR07	1980s	0.02	0	0.84	0	0.14
-CR08	1980s	0.02	0	0.84	0	0.14
-CR09	1980s	0.04	0.01	0.9	0	0.05
-CR10	1980s	0.02	0.07	0.83	0	0.08
-CR11	1980s	0.01	0.02	0.9	0	0.06
-CR02	1990s	0	0	0.06	0.07	0.87
-CR05	1990s	0	0	0.72	0.07	0.21
-CR03	1990s	0	0	0.35	0.06	0.59
-CR04	1990s	0.01	0	0.39	0.49	0.11
-CR06	1990s	0	0	0.33	0.09	0.58
-CR07	1990s	0	0	0.52	0.06	0.41
-CR08	1990s	0	0	0.57	0.31	0.12
-CR09	1990s	0	0	0.55	0.27	0.18
-CR10	1990s	0	0	0.74	0	0.26
-CR11	1990s	0	0	0.69	0.08	0.24
-CR02	2000s	0	0	0	0	1
-CR05	2000s	0	0	0.14	0.14	0.71
-CR03	2000s	0	0	0.2	0.05	0.75
-CR04	2000s	0	0	0.37	0.32	0.31
-CR06	2000s	0	0	0.09	0	0.91
-CR07	2000s	0	0	0.39	0.03	0.58
-CR08	2000s	0	0	0.6	0.22	0.18
-CR09	2000s	0	0	0.67	0.15	0.17
-CR10	2000s	0	0	0.41	0.2	0.39
-CR11	2000s	0	0	0.47	0.02	0.51
-CR02	1940s	0.31	0.17	0.31	0.14	0.07
-CR05	1940s	1	0	0	0	0
-CR03	1940s	1	0	0	0	0
-CR04	1940s	1	0	0	0	0
-CR06	1940s	1	0	0	0	0
-CR07	1940s	1	0	0	0	0
-CR08	1940s	1	0	0	0	0
-CR09	1940s	1	0	0	0	0
-CR10	1940s	1	0	0	0	0
-CR11	1940s	1	0	0	0	0
-CR02	2010s	0	0	0	0	1
-CR05	2010s	0	0	0.14	0.14	0.71
-CR03	2010s	0	0	0.2	0.05	0.75
-CR04	2010s	0	0	0.37	0.32	0.31
-CR06	2010s	0	0	0.09	0	0.91
-CR07	2010s	0	0	0.39	0.03	0.58
-CR08	2010s	0	0	0.6	0.22	0.18
-CR09	2010s	0	0	0.67	0.15	0.17
-CR10	2010s	0	0	0.41	0.2	0.39
-CR11	2010s	0	0	0.47	0.02	0.51
+Dependency=Location Region	Dependency=Vintage	"Option=Wood Stud, Uninsulated"	"Option=Wood Stud, R-7"	"Option=Wood Stud, R-11"	"Option=Wood Stud, R-15"	"Option=Wood Stud, R-19"

```

Comment:
I think we want to remove all double quotations around Option columns.

Reply:
I was hoping you'd say that. For this PR I'll only change the files I've already touched.